integreat_cms/cms/migrations/0112_remove_poi_email_remove_poi_phone_number_and_more.py

```diff
@@ -0,0 +1,55 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from django.db import migrations
+
+if TYPE_CHECKING:
+    from django.apps.registry import Apps
+
+
+def create_primary_contact(apps: Apps) -> None:
+    """
+    Create primary contact from the poi contact data
+
+    :param apps: The configuration of installed applications
+    """
+    POI = apps.get_model("cms", "POI")
+    Contact = apps.get_model("cms", "Contact")
+    for poi in POI.objects.all():
+        if poi.email or poi.phone_number or poi.website:
```

Comment:
We should probably check here whether there is already a primary contact for the POI, to go safer with the constraint that there can only be one single primary contact for each POI.

Reply:
I included the check for it, but I am not sure whether I was able to really check it :see_no_evil: